Integrations/integration-Okta.yml

```diff
@@ -0,0 +1,509 @@
+        } else {
+          err = resp.Status;
+          if (resp.Body) {
+              err += '\n' + resp.Body;
+          }
+          throw err;
+        }
+    };
+
+    var getRaw = function(args, params, uri) {
+        url = fixUrl(params.url);
+        url = url + uri;
+         res = http(
+            url,
+            {
+                Method: 'GET',
+                Headers: {
+                    Authorization: ['SSWS ' + params.apitoken]
+                }
+            },
+            true
+        );
+        return res;
+    };
+
+    var post = function(args, params, uri, body) {
+        url = fixUrl(params.url);
+        url = url + uri;
+        readyBody = null;
+        if (body) {
+            readyBody = JSON.stringify(body);
+        }
+        res = http(
+            url,
+            {
+                Method: 'POST',
+                Headers: {
+                    Authorization: ['SSWS ' + params.apitoken],
+                    Accept: ['application/json'],
+                    'Content-Type': ['application/json']
+                },
+                Body: readyBody
+            },
+            true
+        );
+        return parseResponse(res);
+    };
+
+
+    var get = function(args, params, uri) {
+        raw = getRaw(args, params, uri);
+        res =  parseResponse(raw);
+        if (res && res.length && res.length > 1 ) {
+            return { result: res };
+        } else {
+            return res;
+        }
+    };
+
+    var getUserId = function(args, params) {
+        username = args.username;
+        uri = '/api/v1/users?filter=profile.login+eq+"'+encodeURIComponent(username)+'"';
+        res = get(args, params, uri);
+        if (res && res.length && res.length == 1) {
+            return res[0].id;
+        } else {
+            throw 'Unable to find userId ' + username + '\n' + JSON.stringify(res);
+        }
+    };
+
+    var buildProfile = function(args) {
+        var profile = {};
+        var keys = Object.keys(args);
+        for (var i = 0 ; i < profileArgs.length; i++) {
+            if (args[profileArgs[i]]) {
+                profile[profileArgs[i]] = args[profileArgs[i]];
+            }
+        }
+        return profile;
+    };
+
+    /*
+    {
+        "password": {
+          "value": "tlpWENT2m"
+        },
+        "recovery_question": {
+          "question": "Who's a major player in the cowboy scene?",
+          "answer": "Cowboy Dan"
+        },
+        "provider": {
+          "type": "OKTA",
+          "name": "OKTA"
+        }
+    }
+    */
+    var buildCredentials = function(args) {
+      var creds = {};
+      if (args.password) {
+          var  pswd = { value: args.password };
+          creds.password = pswd;
+      }
+      if (args.passwordQuestion || args.passwordAnswer) {
+          var recovery = {
+              question: args.passwordQuestion,
+              answer: args.passwordAnswer
+          };
+          creds.recovery_question = recovery;
```

Comment:
Why not in a single line? 
creds.recovery_question = {
             question: args.passwordQuestion,
             answer: args.passwordAnswer
         };